src/configure/templates/pythonDjango.yml

```diff
@@ -0,0 +1,58 @@
+# Python Django
```

Comment:
we are not using any azure resource in this template, so azure questions should not be asked. Is this handled in our getInputs logic ?

Reply:
I've handled this, thanks